tests/test_google_sheets_pagination.py

```diff
@@ -57,28 +57,42 @@ def test_run(self):
                 # verify that we can paginate with all fields selected
                 self.assertGreater(record_count_by_stream.get(stream, 0), self.API_LIMIT)
 
+                if stream == "Pagination":
+                    # verify the data for the "Pagination" stream is free of any duplicates or breaks by checking
+                    # our fake pk value ('id')
+                    # THIS ASSERTION CAN BE MADE BECAUSE WE SETUP DATA IN A SPECIFIC WAY. DONT COPY THIS
+                    self.assertEqual(list(map(str, (range(1, 239)))), fake_pk_list)
+    
+                    # verify the data for the "Pagination" stream is free of any duplicates or breaks by checking
+                    # the actual primary key values (__sdc_row)
+                    self.assertEqual(list(range(2, 240)), actual_pk_list)
+                else:
+                    # Setup max rows to check and null rows to check as per the testcase
+                    max_range = 1007
+                    if stream == "sadsheet-pagination": 
```

Comment:
The other whitespace cleanup

```suggestion
                    if stream == "sadsheet-pagination":
```